src/ansys/simai/core/api/training_data.py

```diff
@@ -65,3 +65,8 @@ def compute_training_data(self, training_data_id: str) -> None:
 
     def get_training_data_subset(self, project_id: str, training_data_id: str) -> Dict[str, Any]:
         return self._get(f"projects/{project_id}/data/{training_data_id}/subset")
+
+    def put_training_data_subset(self, project_id: str, training_data_id: str, subset: str) -> None:
```

Comment:
nit: though it's not a practice that is followed in this module, a docstring or/and args explanation would be good.